packages/node/src/network/network.ts

```diff
@@ -192,29 +180,66 @@ export class Network implements INetwork {
 
     this.closed = true;
   }
+
   async getEnr(): Promise<SignableENR | undefined> {
     return this.peerManager["discovery"]?.discv5.enr();
   }
+
+  async getMetadata(): Promise<ts.Metadata> {
+    return {
+      seqNumber: this.metadata.seqNumber,
+      mempoolnets: this.metadata.mempoolnets,
+    };
+  }
+
+  get discv5(): Discv5Worker | undefined {
+    return this.peerManager["discovery"]?.discv5;
+  }
+
+  get localMultiaddrs(): Multiaddr[] {
+    return this.libp2p.getMultiaddrs();
+  }
+
   getConnectionsByPeer(): Map<string, Connection[]> {
     return getConnectionsMap(this.libp2p.connectionManager);
   }
+
   getConnectedPeers(): PeerId[] {
     return this.peerManager.getConnectedPeerIds();
   }
-  hasSomeConnectedPeer(): boolean {
-    return false;
+
+  getConnectedPeerCount(): number {
+    return this.peerManager.getConnectedPeerIds().length;
   }
 
   /* List of p2p functions supported by Bundler */
-  async publishUserOp(_userOp: ts.UserOp): Promise<void> {}
+  async publishUserOpWithEntryPoint(

```

Comment:
Pls rename the method to be plural `publishUserOpsWithEntryPoint`

Reply:
done